README.md

```diff
@@ -1,7 +1,8 @@
 CLI utils
 =========
 
-[![Build Status](https://travis-ci.org/ome/ansible-role-cli-utils.svg)](https://travis-ci.org/ome/ansible-role-cli-utils)
+[![Actions Status](https://github.com/ome/ansible-role-cli-utils/workflows/Molecule/badge.svg)](https://github.com/ome/ansible-role-cli-utils/actions)
+[![Ansible
```

Comment:
cut-n-paste error or unfinished?

Reply:
:+1: thanks for spotting. Fixed in e9748b2